app/app.go

```diff
@@ -195,6 +203,33 @@ func ProjectUpAndWait(p *project.Project, waiter options.Waiter, c *cli.Context)
 		return nil
 	}
 
+	if c.Bool("prune") {
+		url := c.GlobalString("url")
+		accessKey := c.GlobalString("access-key")
+		secretKey := c.GlobalString("secret-key")
+
+		apiClient, err := client.NewRancherClient(&client.ClientOpts{
+			Timeout:   time.Second * 30,
+			Url:       url,
+			AccessKey: accessKey,
+			SecretKey: secretKey,
+		})
+		if err != nil {
+			return err
+		}
+
+		for _, serviceId := range p.GetDeleteServiceIds() {
+			service, err := apiClient.Service.ById(serviceId)
+			if err != nil {
+				return err
+			}
+			err = apiClient.Service.Delete(service)
+			if err != nil {
+				return err
+			}
+		}
+	}
```

Comment:
Are you just deleting all services in the stack? If so, I don't think this is going to be an acceptable solution. The logic probably needs to exist in the p.Create or p.Up function and intelligently resolve the diff between new and old services.

Reply:
Oh, I see GetDeleteServiceIds is a new function you impled that does the diff...thinking